templating/twig_extension_repository.rst

```diff
@@ -0,0 +1,93 @@
+.. index::
+   single: Using the Twig Extensions Repository
+
+How to Use the Twig Extensions Repository
```

Comment:
Unfortunately, "The official Twig extensions" isn't very helpful either, rather misleading...